open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -0,0 +1,406 @@
+{
+ "cells": [
```

Comment:
Please follow this to add the license section and acknowledge section. https://github.com/open-academy/machine-learning/blob/main/open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-advanced.ipynb

Reply:
done.